.changeset/pretty-tools-bathe.md

```diff
@@ -0,0 +1,6 @@
+---
+'@api3/airnode-deployer': patch
```

Comment:
I would say this should be `minor` instead? `patch` should just be for bug fixes

Reply:
I will update it :)